TAC/Meetings/2022/2022-09-22/CCC-TAC-Minutes-2022-09-22.md

```diff
@@ -0,0 +1,130 @@
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+## Agenda and Minutes
+
+### Welcome
+   * Dave Thaler (DT) opened the call at 7:05am PT
+   * DT welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+   * Kurt Taylor (KT) captured the meeting minutes
+   * DT reviewed the agenda and asked for any additions
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+   * Accenture - Giuseppe Giordano
+   * Ant Group - Hongliang Tian (Tate)
+   * Arm - Thomas Fossati + / Michael Lu
+   * Google - Cfir Cohen  / Catalin Sandu +
+   * Huawei - Zhipeng (Howard) Huang +
+   * Intel - Dan Middleton + / Simon Johnson
+   * Meta - Eric Northup + / Shankaran Gnanashanmugam
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann +  / Dimitrios Pendarakis
+" + " indicates attending
+
+***Other Attendees***
+
+   * Jordan Brandt (Inpher)
+   * Mike Bursell (Profian)
+   * XiaoFeng Wang (Indiana University)
+   * Alec Fernandez (Microsoft)
+   * Eric Voit (Cisco)
+   * Francois-Xavier Marseille (THALES)
+   * Jethro Beekman (Fortanix)
+   * Matthieu Legre (Cysec)
+   * Naveen Cherukuri (NVIDIA)
+   * Nick Vidal (Profian)
+   * Penglin Yang (China Mobile)
+   * Steve Van Lare (Anjuna)
+   * Mark Novak (JP Morgan Chase)
+   * Mona Vij (Intel)
+   * Xinxin Fan (IoTex)
+   * Ahmad Atamli (NVIDIA)
+   * Hesham ElBakoury (Consultant)
+   * Ravi Sahita (Rivos)
+   * Drasko Draskovic (Ultraviolet)
+   * Ananya Garg (Microsoft)
+   * Helen Lau (Linux Foundation)
+   * Kurt Taylor (Linux Foundation)
+
+
+**Meeting minute approvals**
+
+Proposed: That the minutes of the September 8, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. Catalin Sandu abstained.
+
+
+**TAC Tech Talks**
+
+   * NSF Center for Distributed Confidential Computing (CDCC) - Prof. Xiaofeng Wang
+       * Mark Novak - Possible collaboration with new Governance (GRC) SIG
+   * MPC + TEE - Jordan Brandt
+       * MPC Alliance Privacy Summit, Oct 5 (virtual only) [https://www.mpcalliance.org/](https://www.mpcalliance.org/)
```

Comment:
Wasn't there an action item or recommendation here?   I thought there was something about followup with the Veracruz project?

Reply:
Jordan Brandt mentioned that MPC Alliance members do work on opensource projects but that the MPC Alliance does not sponsor or focus on any opensource projects. 
Dave Thaler stated that CCC does sponsor opensource projects and named Thomas Fossati and Veracruz.  Thomas said that Dereck would be the person to speak to the high degree of overlap of goals of the two projects.
Jordan said that he and Thomas should connect to join forces and that there would probably be interest on the part of MPC alliance members.
MPC alliance summit might be a good place to make connections.
Thomas has action to make introductions